engine/runtime-compiler/src/main/scala/org/enso/compiler/data/BindingsMap.scala

```diff
@@ -37,11 +38,38 @@ case class BindingsMap(
 
   /** Other modules, imported by [[currentModule]].
     */
-  var resolvedImports: List[ResolvedImport] = List()
+  private var _resolvedImports: List[ResolvedImport] = List()
+
+  def definedEntities: List[DefinedEntity] = {
+    ensureConvertedToConcrete()
+    _definedEntities
+  }
+  def currentModule: ModuleReference = {
+    ensureConvertedToConcrete()
+    _currentModule
+  }
+  def resolvedImports: List[ResolvedImport] = {
+    ensureConvertedToConcrete()
+    _resolvedImports
+  }
+  def resolvedImports_(v: List[ResolvedImport]): Unit = {
```

Comment:
minor: rename to `setResolvedImports` for clarity

Reply:
I was trying to follow Scala's conversion on [accessors and mutators](https://docs.scala-lang.org/style/naming-conventions.html#accessorsmutators). Fixed in 2ada503d7d